openstack/compute/v2/extensions/hypervisors/doc.go

```diff
@@ -24,7 +24,28 @@ Example of Show Hypervisor Details with Compute API microversion greater than 2.
 
 Example of Retrieving Details of All Hypervisors
 
-	allPages, err := hypervisors.List(computeClient).AllPages()
+	allPages, err := hypervisors.List(computeClient, nil).AllPages()
+	if err != nil {
+		panic(err)
+	}
+
+	allHypervisors, err := hypervisors.ExtractHypervisors(allPages)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, hypervisor := range allHypervisors {
+		fmt.Printf("%+v\n", hypervisor)
+	}
+
+Example of Retrieving Details of All Hypervisors with Compute API microversion greater than 2.33
```

Comment:
Not clear which microversion end user needs to use. 2.33 or 2.53 (from the acceptance tests)

Reply:
It is confusing, yes. The newly added `ListOpts` includes options that are valid in either 2.33 or 2.53. The title of this example is supposed to denote that microversion 2.33+ is required.